Dockerfile

```diff
@@ -1,14 +1,24 @@
-FROM golang:1.17
+FROM golang:alpine as builder
 ARG IMAGE_TAG
+
 RUN mkdir -p /go/src/github.com/ScoreTrak/ScoreTrak
 WORKDIR /go/src/github.com/ScoreTrak/ScoreTrak
+
 COPY pkg/ pkg/
 COPY cmd/ cmd/
 COPY main.go main.go
 COPY go.mod go.mod
 COPY go.sum go.sum
-RUN go mod download
-RUN go build -o scoretrak -ldflags "-X 'github.com/ScoreTrak/ScoreTrak/pkg/version.Version=${IMAGE_TAG}'"
-RUN chmod +x scoretrak
-ENTRYPOINT ["./scoretrak"]
+
+ENV CGO_ENABLED=0
+RUN go mod download && go build -o scoretrak -ldflags "-X 'github.com/ScoreTrak/ScoreTrak/pkg/version.Version=${IMAGE_TAG}'"
+
+FROM alpine:latest
```

Comment:
let's use stable alpine version, instead of lates

Also, I just to note, CGO_ENABLED is required for gorm, when it is used in conjunction with sqlite. I know at the moment we are more or less interested in using cockroach for resiliency, but in the future, if we decide to go with some minimal container version, CGO_ENABLED might be needed.

For now CGO_ENABLED=0 is fine

Reply:
Interesting, will keep this noted for later reference